game/server/supplyroutes/models.py

```diff
@@ -91,7 +91,7 @@ def for_link(
             #
             # https://reactjs.org/docs/lists-and-keys.html#keys
             # https://github.com/dcs-liberation/dcs_liberation/issues/2167
-            id=uuid.uuid4(),
+            id=UUID(int=a.id.int ^ b.id.int, version=4),
```

Comment:
This is... odd. I've spent enough time trying to reason about whether or not this safe to believe that we should just not do it. Change the id field to a string and just do `f"{a}:{b}"`?

Reply:
Consider it done...